docs: clarify two-way binding explanation in Numeric inputs tutorial (#1685)
sveltejs/svelte.dev: apps/svelte.dev/content/tutorial/01-svelte/06-bindings/01-text-inputs/index.md

<input +++bind:+++value={name}>
```

> This means that changes to the value of `name` update the input, and changes to the input update `name`.

Reviewer: ```suggestion
This means that as well as changes to `name` updating the `<input>`, changes to the `<input>` will update `name`.
```
